tts-evaluation-MCD-DTW.ipynb

```diff
@@ -0,0 +1,433 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f0408502",
+   "metadata": {},
+   "source": [
+    "# Calculating MCD DTW(Mel cepstral distortion - Dynamic time warping)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a749b10",
+   "metadata": {},
+   "source": [
+    "Mel cepstral distortion is an objective measure of speech quality. This notebook calculates MCD between a tts generated audio and a ground truth audio. Closer the mcd value is to zero, closer the simlarity between two audios. MCD of an audio with itself will result in a value of 0.\n",
+    "\n",
+    "MCD DTW is a modification of MCD that works with non aligned audio clips by using dynamic time warping cost matrix.Tthe scale depends on factors such as mel extractor and reduction algorithm(mean of DTW cost or min DTW path cost). These limitations render this metric useless for a comprehensive evaluation, however it is useful for comparing the model convergence. The models trained on similar mels can be compared with respect to ground truth, to get the model with better convergence.\n"
```

Comment:
`The models trained on similar mels can be compared with respect to ground truth, to get the model with better convergence.` - This sentence could be re-worked to make better sense